### PR TITLE
Adds steps  for running perf tests individually on local

### PR DESCRIPTION
**Pull request by kaibocai**
# Description

This PR adds a few elaborations on running perf tests locally. It should help developers to better running the process. 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [X] Code compiles correctly
* [X] Created/updated tests
* [X] Unit tests passing
* [X] End-to-end tests passing
* [X] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [X] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [X] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

File: tests/docs/running-perf-tests.md [hunk 1]
```diff
@@ -140,6 +141,21 @@ make setup-test-env-k6
 make test-perf-all
 ```
 
+You can also run the test apps individually.
```
Comment (ItalyPaleAle) [inline tests/docs/running-perf-tests.md:144]: The best way is to actually use DAPR_PERF TEST that allows passing more than one test:

For example, this runs 2 actors test:

```sh
DAPR_PERF_TEST="actor_id_scale actor_type_scale" make test-perf-all
```

File: tests/docs/running-perf-tests.md [hunk 2]
```diff
@@ -140,6 +141,21 @@ make setup-test-env-k6
...
+# push perf apps docker image to docker hub
+make test-perf-<app-name>
+```
+`<app-name>` can be found at [PERF_TESTS](https://github.com/dapr/dapr/blob/6def7d1b9ffe896b7b06d05128b9cd605d39f939/tests/dapr_tests.mk#L70)
+
+For example, if you only want to run workflows tests, you can run
+```bash
+make test-perf-workflows
+```
+
```
Comment (ItalyPaleAle) [inline tests/docs/running-perf-tests.md:158]: Also a helpful command is "make test-clean" which removes all test data so it's easier to find the new one!

**Review by mukundansundar — APPROVED** [review]
@kaibocai welcome to the community. Thanks for the contribution.

File: tests/docs/running-perf-tests.md (hunk 1 above)
Comment (kaibocai) [reply tests/docs/running-perf-tests.md]: updated as suggested.

File: tests/docs/running-perf-tests.md (hunk 2 above)
Comment (kaibocai) [reply tests/docs/running-perf-tests.md:166]: updated as suggested.

**Review by mukundansundar — APPROVED** [review]

**Pull request merged**